index.js

```diff
@@ -13,18 +11,6 @@ var runner = new Runner({
 	match: opts.match
 });
 
-// check if the test is being run without AVA cli
-var isForked = typeof process.send === 'function';
-
-if (!isForked) {
-	var fp = path.relative('.', process.argv[1]);
-
-	console.log();
-	console.error('Test files must be run with the AVA CLI:\n\n    ' + chalk.grey.dim('$') + ' ' + chalk.cyan('ava ' + fp) + '\n');
-
-	process.exit(1); // eslint-disable-line
-}
```

Comment:
We'd better move it to its own utility file. `wrap-send.js` does not sound like a good place for it either. I'm requiring a module to wrap a function, but under the hood it checks if the process was forked?